apps/testingaccessibility/src/components/portable-text/index.tsx

```diff
@@ -242,20 +242,32 @@ const PortableTextComponents: PortableTextComponentsProps = {
     },
     bodyImage: ({value}: BodyImageProps) => <BodyImage value={value} />,
     code: ({value}: CodeProps) => {
-      const {language, code, highlightedLines} = value
-
+      const {language, code, highlightedLines, filename: label} = value
       return (
-        <Refractor
-          language={
-            language
-              ? Refractor.hasLanguage(language)
-                ? language
-                : 'javascript'
-              : 'javascript'
-          }
-          value={code}
-          markers={highlightedLines}
-        />
+        <>
+          <pre
+            role="region"
+            aria-label={label ? label : 'code sample'}
```

Comment:
![code block label in sanity studio](https://user-images.githubusercontent.com/25487857/178244454-bcc7c19d-92c1-46bc-bdf3-5b3b167e2c90.png)